source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+
+Optionally, if you already have a |CL| system set up that you want to install
+the telemetry backend server components to, or to a virtual |CL| machine, you
+need to have a user account that has sudo privileges and the following
+software bundles already installed:
+
+* dev-utils - has the :command:`git` command
+* telemetrics - has the required telemetry applications, see the 
+  :ref:`telemetry-enable` guide for installing and enabling telemetrics.
+
+To determine the software bundles already installed, enter the
+:command:`sudo swupd bundle-list` command to list the current bundles
+installed and then run :command:`sudo swupd bundle-add dev-utils telemetrics`
+if either one is not installed.
+
+Download the clearlinux/telemetrics-backend git repository
+==========================================================
+
+With all prerequisite software bundles installed and logged in with your
+administrative user, from your $HOME directory, run :command:`git` to clone
+the :guilabel:`telemetrics-backend` repository into the
+:file:`$HOME/telemetrics-backend` directory:
+
+.. code-block:: console
+
+   git clone https://github.com/clearlinux/telemetrics-backend
+
+.. note::
+   You may need to set up the :guilabel:`https_proxy` environment variable
+   if you have issues reaching github.com.
+
+Run the deploy.sh script to install the backend server
+======================================================
+
+Change your current working directory to :file:`telemetrics-backend/scripts`
+and then run the :command:`./deploy.sh -h` to see the list of options for the
+:command:`deploy.sh` script:
+
+.. code-block:: console
+
+   cd telemetrics-backend/scripts
+   ./deploy.sh -h
+  
+   Deploy snapshot of the telemetrics-backend
+
+     -a    Perform specified action (deploy, install, migrate, resetdb,
+           restart, uninstall; default: deploy)
+     -d    Distro to deploy to (ubuntu, centos or clr; default: ubuntu)
+     -h    Print these options
+     -H    Set domain for deployment (only accepted value is "localhost" for
+           now)
+     -r    Set repo location to deploy from
+           (default: https://github.com/clearlinux/telemetrics-backend)
+     -s    Set source location (default: "master" branch from git repo)
+     -t    Set source type (tarball, or git; default: git)
+     -u    Perform complete uninstallation
+
+The :command:`deploy.sh` is a bash shell script that allows you to perform the
+following actions:
+
+* deploy - install a complete instance of the telemetrics backend
+  server and all required components. This is the default action if no
+  :command:`-a` argument is given on the command line.
+* install - installs and enables all required components for the telemetrics
+  backend server.
+* migrate - migrate database to new schema.
+* resetdb - reset the database.
+* restart - restart the nginx and uWSGI services.
+* uninstall - uninstall all Ubuntu packages if the distribution is Ubuntu.
```

Comment:
While the description here is technically correct, this document is describing how to install telemetrics-backend on Clear Linux, so it may be more useful to say something along these lines: "uninstall required components added by install (currently a no-op on Clear Linux)"

Reply:
I concur, we should include the Clear Linux specific information and not the generic one.

Reply:
So I changed the wording for uninstall and then added a note.  Let me know your thoughts on what I put in there